crud/select/plan.lua

```diff
@@ -145,6 +181,12 @@ function select_plan.new(space, conditions, opts)
     local scan_value
     local scan_condition_num
 
+    local space_field_positions = {}
+
+    for i, field in ipairs(space_format) do
+        space_field_positions[field.name] = i
+    end
```

Comment:
Looks the same as `utils.get_format_fieldno_map()`.

Reply:
Thanks, used the util